microcosm_pubsub/registry.py

```diff
@@ -90,8 +93,25 @@ def __getitem__(self, media_type):
         Create a codec or raise KeyError.
 
         """
-        schema_cls = self.mappings[media_type]
-        return PubSubMessageCodec(schema_cls(strict=self.strict))
+        try:
+            schema_cls = self.mappings[media_type]
+            if isinstance(schema_cls, string_types):
```

Comment:
If we use `@handles(created("Foo"))`, we will register a string as the media type; in this case, we assume the URI schema.